src/conn/conn_layered.c

```diff
@@ -1136,6 +1123,9 @@ __wti_disagg_conn_config(WT_SESSION_IMPL *session, const char **cfg, bool reconf
         WT_ERR(__wt_config_gets(session, cfg, "disaggregated.lose_all_my_data", &cval));
         if (cval.val != 0)
             F_SET(&conn->disaggregated_storage, WT_DISAGG_NO_SYNC);
+
+        /* Start the layered table manager thread. */
```

Comment:
I realise this was just moved, but the comment was a bit pointless in its old location too :-)

Could you remove it please?